README.md

```diff
@@ -156,6 +132,33 @@ git-credential-manager-core unconfigure
 rm $(command -v git-credential-manager-core)
 ```
 
+#### Option 2: Install from source helper script
+
+1. Ensure `curl` is installed:
+
+   ```shell
+   curl --version
+   ```
+
+   If `curl` is not installed, please use your distribution's package manager
+   to install it.
+
+1. Download and run the script:
+
+   ```shell
+   curl -LO https://raw.githubusercontent.com/GitCredentialManager/git-credential-manager/main/src/linux/Packaging.Linux/install-from-source.sh &&
+   sh ./install-from-source.sh &&
+   git-credential-manager-core configure
+   ```
+
+   **Note:** You will be prompted to enter your credentials so that the script
+   can download GCM's dependencies using your distribution's package
+   manager.
+
+To uninstall:
+
+[Follow these instructions](docs/faq.md#i-used-the-install-from-source-script-to-install-gcm-on-my-linux-distribution-now-how-can-i-uninstall-gcm-and-its-dependencies) for your distribution.
```

Comment:
I'd say link to a separate markdown file with the 'from source' uninstall instructions rather than the FAQ.